syz-ci/manager.go

```diff
@@ -256,6 +256,17 @@ loop:
 	log.Logf(0, "%v: stopped", mgr.name)
 }
 
+func (mgr *Manager) archiveHeadCommit(commit string) {
+	if mgr.cfg.GitArchive == "" || mgr.mgrcfg.DisableGitArchive {
+		return
+	}
+	log.Logf(0, "%v: archiving commit %s", mgr.name, commit)
+	if err := mgr.repo.ArchiveHeadCommit(mgr.cfg.GitArchive, commit); err != nil {
+		log.Logf(0, "%v: failed to archive commit %s from repo %s: %s",
```

Comment:
We need to surface it to dashboard or our GCP monitoring. Nobody is monitoring these logs.

Reply:
Checked what we alert. This one should be changed to log.Errorf.
It should be enough.

Reply:
mgr.Errorf